### DISCUSSION TITLE
As shown in the screenshot, Backblaze B2 shows two files which are not shown in Cyberduck in the same directory.

### DISCUSSION THREAD

**Discussion by blumorrell** [iterate-ch/cyberduck#16316] [Q&A] [answered]
This ticket system is to report bugs and feature requests. For support, visit the [help page](https://cyberduck.io/help/) first.

 * If you have trouble connecting to a server or your login credentials are not valid, try to resolve the issue with the assistance of your hosting service provider. Any such ticket will be closed with a resolution of _thirdparty_.

**Describe the bug**
As shown in the screenshot, Backblaze B2 shows two files which are not shown in Cyberduck in the same directory.

**To Reproduce**
Steps to reproduce the behavior: n/a

**Expected behavior**
Cyberduck should show the files

**Screenshots**
Attached

**Desktop (please complete the following information):**
 - OS: MacOS 
 - Version 14.5 (23F79)

**Log Files**
Please include the [log file](https://docs.cyberduck.io/cyberduck/support/#logging-output). You can reach the logging output in _Preferences → Connection_.

**Additional context**

**Comment by blumorrell** [comment]
[cyberduck.log](https://github.com/user-attachments/files/16589639/cyberduck.log)

**Comment by blumorrell** [comment]
<img width="1435" alt="backblaze screenshot" src="https://github.com/user-attachments/assets/e4263800-a8ae-4377-a40a-e29aab048cce">

**Comment by dkocher** [comment, marked as answer]
The asterix displayed behind the filename suggests these file are marked as _hidden_ and therefore not displayed by default.  Choose _View → Show Hidden Files_ as indicated in the [documentation](https://docs.cyberduck.io/protocols/b2/#file-versioning).
